### PR TITLE
chore(user): migrate route to application controller

### LINKED ISSUE #69: Migrate user operation to Dashboard
Since profile update comes under the dashboard, move it to the dashboard.controller.ts file.

The function in dashboard.controller.ts should use the user service.

https://github.com/UniAuth/uniauth-backend/blob/591c0553288e45dfd46cacd5f1d622c68f281560/src/user/user.controller.ts#L69-L91

### PR DESCRIPTION
**Pull request by kdsinghcoder**
# Issue Related to PR
- fixes #69 , #71

# Additional info if necessary
- any additional information

# Checklist
- [x] Tests
- [x] Resolve Conflicts

# Screenshots

Original    |   Updated
:----------------: | :----------------:
image | image

### CODE REVIEW

File: src/dashboard/dashboard.controller.ts [hunk 1]
```diff
@@ -87,6 +89,20 @@ export class DashboardController {
...
+    return res.render('profile/edit.hbs', { user });
+  }
+
+  @Post('/:id/edit')
+  @UseGuards(JwtAuthGuard)
+  async PostEditForm(@Res() res: Response, @Param('id') id: string , @Body() updateUserDto: UpdateUserDto) {
+    await this.userService.update(id , updateUserDto)
+    return res.redirect('/dashboard/');
+
+
```
Comment (YashKumarVerma) [inline src/dashboard/dashboard.controller.ts:105]: missing }

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Review by kdsinghcoder — COMMENTED** [review]
Fixed

**Review by kdsinghcoder — COMMENTED** [review]
fixed required changes.

**Review by YashKumarVerma — APPROVED** [review]

**Comment by YashKumarVerma** [comment]
thanks !

**Pull request merged**